cli/cook/subcommands/kill.py

```diff
@@ -91,7 +91,7 @@ def kill_entities(query_result, clusters):
 def kill(clusters, args, _):
     """Attempts to kill the jobs / instances / groups with the given UUIDs."""
     guard_no_cluster(clusters)
-    uuids = strip_all(args.get('uuid'))
+    uuids = parse_entity_refs(args.get('uuid'))
```

Comment:
Should we have an alias for the `uuid` option? Maybe `entity`?

Reply:
What do you mean by an alias?

Reply:
I forgot that these are positional arguments. I was thinking you had to do something like `cs kill --uuid $x`, in which case supporting `cs kill --entity $x` might be good too.

Reply:
Got it, yeah, they are positional